internal/p4constants/p4constants.go

```diff
@@ -111,6 +111,128 @@ const (
 	PacketMetaPacketOut uint32 = 75327753
 	PacketMetaPacketIn  uint32 = 80671331
 	// Meters
-	MeterPreQosPipeAppMeter     uint32 = 338231090
-	MeterPreQosPipeSessionMeter uint32 = 347593234
+	MeterPreQosPipeAppMeter         uint32 = 338231090
+	MeterSizePreQosPipeAppMeter     uint64 = 1024
+	MeterPreQosPipeSessionMeter     uint32 = 347593234
+	MeterSizePreQosPipeSessionMeter uint64 = 1024
+)
+
+var (
+	TableIDToNameMap = map[uint32]string{
```

Comment:
In principle, we should have immutable maps and lists here. Go, however, does not allow maps and lists to be declared as `const`. We could use this, though : https://github.com/benbjohnson/immutable

Reply:
One trivial workaround I have seen is "functions returning constants". That way nobody can modify the master copy, just the local clone.

Not Go, but shows the principle: https://github.com/pins/pins-infra/blob/master/p4_pdpi/p4_runtime_session.h#L67-L73

Reply:
is the repository private? I get a 404

Reply:
oh, yes it is. The playgound link is better anyway.

Reply:
Done